docs/020-deployment/howtos/080-configure-dns.md

```diff
@@ -98,38 +85,39 @@ nano Cluster-Current-values.yaml
 ![configure-dns-1.png](../../assets/images/ama/howtos/configure-dns-1.png)
 4. Edit the value of the host name to reflect your main domain. If you have 3 environments (development, test, and production), update the value of the host name for each environment (`dev.mycompany.com`, `test.mycompany.com`, `prod.mycompany.com.`).
 5. Find the **TLS hosts** section. The example of the section is shown below.
-```
+```yaml
+global:
+  ingress:
     tls:
       hosts:
-       departmentX.20.0.189.11.sslip.io
-       app.20.0.189.11.sslip.io
-       clean.20.0.189.11.sslip.io
-       grafana.20.0.189.11.sslip.io
-       prometheus.20.0.189.11.sslip.io
-       alertmanager.20.0.189.11.sslip.io
-       '*.20.0.189.11.sslip.io'
-```
-6. Replace all instances of `20.0.189.11.sslip.io` to `mycopmany.com`. The example of the updated section is shown below.
+      - departmentX.20.0.189.11.sslip.io
+      - app.20.0.189.11.sslip.io
+      - clean.20.0.189.11.sslip.io
+      - '*.20.0.189.11.sslip.io'
 ```
+6. Replace the hosts section with the below. The example of the updated section is shown below.
+```yaml
+global:
+  ingress:
     tls:
       hosts:
-       deparmentX.mycopmany.com
-       app.mycopmany.com
-       clean.mycopmany.com
-       grafana.mycopmany.com
-       prometheus.mycopmany.com
-       alertmanager.mycopmany.com
-       '*.mycopmany.com
+      - departmentX.mycompany.com
+      - app.mycompany.com
+      - clean.mycompany.com
+      - '*.mycompany.com'
 ```
-7. If you are using multiple environments (development, test, and production), update the TLS hosts section for each environment:
+
+7. If you are using multiple environments (development, test, and production), update the TLS hosts section for each environments `values.yaml` file:
    - Development – replace all instances of `20.0.189.11.sslip.io` to `dev.mycompany.com`
    - Test – replace all instances of `20.0.189.12.sslip.io` to `test.mycompany.com`
    - Production – replace all instances of `20.0.189.13.sslip.io` to `prod.mycompany.com`
-8. Save the file.
+8. Save the file
 9. Apply changes from the local file to the CluedIn cluster by running the following command:
 ```
-helm upgrade -i cluedin-platform cluedin/cluedin-platform  -n cluedin --create-namespace  --values Cluster-Current-values.yaml --set application.system.runDatabaseJobsOnUpgrade=false 
+helm upgrade -i cluedin-platform cluedin/cluedin-platform  -n cluedin --create-namespace  --values Cluster-Current-values.yaml
 ```
 
 After a short time, you'll see the confirmation of your update in the console. CluedIn is now configured to use your new DNS address.
 ![configure-dns-2.png](../../assets/images/ama/howtos/configure-dns-2.png)
+
+Note: This will use the LetsEncrypt service in the cluster to do a http request to validate the certificate. If you would like to use a self-provided certificate, please review the Configure [`TLS Certificates`](/deployment/infra-how-tos/configure-certificates) page
```

Comment:
```suggestion
**Note:** This will use the LetsEncrypt service in the cluster to do an HTTP request to validate the certificate. If you would like to use a self-provided certificate, please review the Configure [`TLS Certificates`](/deployment/infra-how-tos/configure-certificates) page
```